Lib/email/_policybase.py

```diff
@@ -307,7 +307,7 @@ def header_source_parse(self, sourcelines):
 
```

Comment:
s/remainder of the first line, joining all subsequent lines together/remainder of the first line joined with all subsequent lines/